README.md

```diff
@@ -93,6 +93,10 @@ After a successful build, you should have a `glslc` executable somewhere under
 the `$BUILD_DIR/glslc/` directory, as well as a `libshaderc` library somewhere
 under the `$BUILD_DIR/libshaderc/` directory.
 
+The default behavior on MSVC is to link with the static CRT. If you would like
```

Comment:
I think that that statement is not sufficient. What really matters to the user is that they link the static CRT since this is something that has to match between a user's program and libshaderc. The statement "we statically link all libraries and executables" does not necessarily convey this information in a clear way. This is especially true since the flag is -DSHADERC_ENABLE_SHARED_CRT.